### ISSUE TITLE
Feature Request - Locking Value

### ISSUE THREAD

**Issue by Merijeek** [comfyanonymous/ComfyUI#367]
I have been playing around, and one thing I've found is that values change without my noticing. Now, I'm 100% certain that this is because of sloppy mouse work. 

But it might be nice to be able to lock values (CFG, steps, etc) so that they can't accidentally be changed. Just add a lock icon that can be toggled to avoid accidental changes.

**Comment by WASasquatch** [comment]
> I have been playing around, and one thing I've found is that values change without my noticing. Now, I'm 100% certain that this is because of sloppy mouse work.
> 
> But it might be nice to be able to lock values (CFG, steps, etc) so that they can't accidentally be changed. Just add a lock icon that can be toggled to avoid accidental changes.

I agree. You can be scrolling one value, and apparently hover other and change them as well. Sometimes clicking a value with increment it because the increment/deincrement is the whole right/left side of the input, and not localized to the arrows.

**Comment by comfyanonymous** [comment]
>Sometimes clicking a value will increment it because the increment/deincrement is the whole right/left side of the input

I updated litegraph and it's fixed.
